test/utils.py

```diff
@@ -5,14 +5,15 @@
 
 import ITR
 from pint import Quantity
-from pint_pandas import PintType
```

Comment:
Fine for now.  As raised in the initial merge, this file is duplicated from ITR.  We need to rationalize whether we need our own utils file for the examples test cases, or whether we should use ITR.utils.